File: example/AnyJob.NetCore.Demo/Instance.cs
using System;
using System.Collections.Generic;
using System.Linq;
namespace AnyJob.NetCore.Demo
{
    public class Instance
    {
        public int Add(int num1, int num2)
        {
            return num1 + num2;
        }

        public string Concat(string a, string b)
        {
            return a + b;
        }
        public void Hello(string name)
        {
            Console.WriteLine($"Hello,{name}");
        }

        public List<PersonInfo> Merge(List<PersonInfo> persons, PersonInfo other)
        {
            var result = new List<PersonInfo>();

            result.AddRange(persons ?? Enumerable.Empty<PersonInfo>());
            if (other != null)
            {
                result.Add(other);
            }
            return result;
        }
    }
}

File: example/AnyJob.Java.Demo/src/main/java/demo/Instance.java
package demo;

import java.util.ArrayList;
import java.util.List;

public class Instance {
    public int add(int num1, int num2) {
        return num1 + num2;
    }

    public String concat(String a, String b) {
        return a + b;
    }

    public void hello(String name) {
        System.out.println("Hello," + name);
    }

    public List<Person> merge(List<Person> persons, Person other) {
        List<Person> result = new ArrayList<Person>();
        if (persons != null) {
            result.addAll(persons);
        }
        if (other != null) {
            result.add(other);
        }
        return result;
    }
}
